### DISCUSSION TITLE
Change Folder Name in the Sidebar

### DISCUSSION THREAD

**Discussion by nicolovitelli** [squidfunk/mkdocs-material#7674] [Q&A]
Hi, I'm trying to change a Folder's name without changing the actual name.

I've got this folder "metodi" in my docs directory:
![image](https://github.com/user-attachments/assets/bbc95f1b-f798-4f7c-997c-ed3fac54523b)

Obviously, in the sidebar it's being showed like this:
![image](https://github.com/user-attachments/assets/0b38c833-77eb-4638-952e-edddc9cf482f)

How can I change the folder name in the sidebar without having to change the actual folder's name?
I've already tried adding `nav ` in `mkdocs.yml` but I don't think it's the correct solution because I would have to manually add every file under the Metodi folder. 

Thank you for the help

**Comment by alexvoss** [comment]
The [awesome pages plugin](https://github.com/lukasgeiter/mkdocs-awesome-pages-plugin) might be what you are looking for.
